content/tutorials/iis.md

```diff
@@ -12,42 +12,66 @@ aliases: ["/docs/grafana/latest/tutorials/iis/"]
 
 # Use IIS with URL Rewrite as a Reverse Proxy
 
-If you want Grafana to be a subpath or subfolder under a website in IIS then the URL Rewrite module for ISS can be used to support this.
+If you want Grafana to be a subpath/subfolder under a website in IIS then the Application Request Routing (ARR) and URL Rewrite modules for ISS can be used to support this.
 
 Example:
 
-- Parent site: http://localhost:8080
+- Parent site: http://yourdomain.com:8080
 - Grafana: http://localhost:3000
 
-Grafana as a subpath: http://localhost:8080/grafana
+Grafana as a subpath: http://yourdomain.com:8080/grafana
+
+Other Examples:
+- The parent site could also look like http://localhost:8080 if the application was only served on the local server.
```

Comment:
```suggestion
- If the application is only served on the local server, the parent site can also look like http://localhost:8080. 
```

Reply:
Served on the local server sounds odd. I am not sure of the exact context, but perhaps we could find a better word?

Reply:
That's the best way to described it so there is no ambiguity.  The objective should not be perfect english (as the last one was not amazing in this regard) it should be to ensure the user can't misinterpret the instructions.